docs/user/alerting/alerting-troubleshooting.asciidoc

```diff
@@ -12,6 +12,32 @@ If your problem isn’t described here, please review open issues in the followi
 
 Have a question? Contact us in the https://discuss.elastic.co/[discuss forum].
 
+[float]
+[[rule-cannot-decrypt-api-key]]
+=== Rule cannot decrypt apiKey
+
+*Problem*:
+
+The rule fails to execute and has an `Unable to decrypt attribute "apiKey"` error.
+
+*Solution*:
+
+This error happens when the `xpack.encryptedSavedObjects.encryptionKey` value used to create the rule does not match the value used during rule execution. Depending on the scenario, there are different ways to solve this problem:
+
+[cols="2*<"]
+|===
+
+| If the value in `xpack.encryptedSavedObjects.encryptionKey` was manually changed, and the previous encryption key is still known.
+| Ensure any previous encryption key is part of the <<encryption-key-rotation, decryption only key>>.
```

Comment:
Is "decryption only key" the correct wording?

Reply:
As discussed offline, I will try `Ensure any previous encryption key is included in the keys used for decryption only.`.